feat: HCaptcha.redrawView (#140)
hCaptcha/HCaptcha-ios-sdk: README.md

    + [Using landscape instead of portrait orientation](#using-landscape-instead-of-portrait-orientation)
    + [SDK Events](#sdk-events)
    + [Disable new token fetch on expiry](#disable-new-token-fetch-on-expiry)

>     + [Change hCaptch frame](#change-hcaptcha-frame)

Reviewer: ```suggestion
    + [Change hCaptcha frame](#change-hcaptcha-frame)
```

   }
```

> ### Change hCaptch frame
> 
> In case if you need to change hCaptcha layout for example after visual challenge appear you can use approach above

Reviewer: ```suggestion
### Change hCaptcha frame

If you are customizing display beyond the defaults and need to resize or change the hCaptcha layout, for example after a visual challenge appears, you can use the following approach  to trigger a redraw of the view:
```

> If you are customizing display beyond the defaults and need to resize or change the hCaptcha layout, for example after a visual challenge appears, you can use the following approach  to trigger a redraw of the view:

Reviewer: ```suggestion
If you are customizing display beyond the defaults and need to resize or change the hCaptcha layout, for example after a visual challenge appears, you can use the following approach to trigger a redraw of the view:
```
